pkgs/servers/mpd/default.nix

```
@@ -2,7 +2,7 @@
 # Darwin inputs
 , AudioToolbox, AudioUnit
 # Inputs
-, curl, libmms, libnfs, liburing, samba
+, libcdio, libcdio-paranoia, curl, libmms, libnfs, liburing, samba
```

Nit: Let's keep the local sort order intact.
```suggestion
, curl, libcdio, libcdio-paranoia, libmms, libnfs, liburing, samba
```